pandas/conftest.py

```diff
@@ -995,6 +1004,14 @@ def _create_series(index):
 }
 
 
+@pytest.fixture
+def f8series_any_simple_index(indices):
```

Comment:
what is the purpose here? this name is impossible to grok

Reply:
Any ideas for a better alternative @jreback?

@jbrockmendel and I already brainstormed a bit [in these comments](https://github.com/pandas-dev/pandas/pull/32046#discussion_r382704106), but couldn't come up anything something better...

Other ideas:
- `series_with_changing_index`
- `series_with_simple_index`
- ...

Reply:
series_with_simple_index is more indicative